src/routes/dashboard.js

```diff
@@ -42,10 +66,84 @@ export const updateUser = async (db, collection, email, dataToBeUpdated) => {
   }
 };
 
+export const getUser = async (db, collection, id) => {
+  try {
+    const userArray = await db.collection(collection).find({
+      id,
+    }).project({
+      _id: 0, username: 1, name: 1,
+    }).toArray();
+    const [user] = userArray;
+    if (user === undefined) {
+      return null;
+    }
+    return user;
+  } catch (e) {
+    return {
+      error: {
+        name: 'Error finding user',
+      },
+    };
+  }
+};
+
+export const getOrg = async (db, collection, id) => {
+  try {
+    const organisationArray = await db.collection(collection).find({
+      id,
+    }).project({
+      _id: 0, name: 1, description: 1, ownerID: 1,
+    }).toArray();
+    const [organisation] = organisationArray;
+    if (organisation === undefined) {
+      return null;
+    }
+    return organisation;
+  } catch (e) {
+    return {
+      error: {
+        name: 'Error finding organisation',
+      },
+    };
+  }
+};
+
 route.get(routes.Home, (req, res) => {
   res.json({ success: true });
 });
 
+route.get(dashboardRoutes.GetUser, async (req, res) => {
+  const db = await getDb();
+  const collection = productionConstants.USERS_COLLECTION;
+  const userID = req.params.id;
+  const user = await getUser(db, collection, userID);
+  if (user === null) {
+    res.json({
+      error: {
```

Comment:
I see that you are making an  object with error key that has other keys. I guess the error object structure is set. So you can make a function such as makeErrorObject that will make the object for you.